server/webapp.js

```diff
@@ -44,7 +47,7 @@ function create(flags) {
   app.use('/status', status);
   app.use('/client', express.static(path.join(base, 'client')));
```

Comment:
the client dir is relative to the chicago-brick root, regardless of if it is run top-level or as a dep.